pkg/ipam/allocator.go

```diff
@@ -96,6 +96,33 @@ func (ipam *IPAM) AllocateIPString(ipAddr, owner string) error {
 }
 
 func (ipam *IPAM) allocateNextFamily(family Family, allocator Allocator, owner string) (result *AllocationResult, err error) {
+	if EnableFixedIP && owner != "loopback" && owner != "health" {
+		// Reusing IP if pinned IP found for this Pod.
+		//
+		// Actually we should first determine if this pod belongs to a sts,
+		// since we only pin IP for sts pods.
+		// But the determine step involves an further call to K8S apiserver.
```

Comment:
It would be nice if you change the above #L104 to But the step to determine involves further call to K8S apiserver.

Reply:
hi @soumynathan , done for that.Thanks!